README.md

```diff
@@ -23,25 +23,49 @@ The documentation can be generated using Sphinx.
 
 # Getting Started
 
-Make sure git submodules are cloned:
+To initialize submodule and setup the CMake build system, from the root of the
+`symbiflow-arch-defs` directory run:
 
 ```
-git submodule init
-git submodule update
+make env
 ```
 
-Install Python libraries:
+At this point a new directory `build` will have been created, which is where
+you can invoke make to build various targets.
+
+To build all demo bitstreams there are 3 useful targets
 
 ```
-# Make sure additional libraries are installed
-sudo apt-get install libxml2-dev libxslt1-dev lib32z1-dev
-pip install -r requirements.txt
+# Build all demo bitstreams, targetting all architectures
+make all_demos
+
+# Build all 7-series demo bitstreams
+make all_xc7
+
+# Build all ice40 demo bitstreams
+make all_ice40
 ```
 
-Run the full suite:
+Specific bitstreams can be built by specifying their target name, followed by
+a suffix specifying the desired output. For example, the LUT-RAM test for the
+RAM64X1D primative is called `dram_test_64x1d`.  Example targets are:
+
 
 ```
-make
+# Just run synthesis on the input Verilog
+make dram_test_64x1d_eblif
+
+# Complete synthesis and place and route the circuit
+make dram_test_64x1d_route
+
+# Create the output bitstream (including synthesis and place and route)
+make dram_test_64x1d_bin
+
+# Convert the generated bitstream back into verilog
+make dram_test_64x1d_bit_v
+
+# Run the verilog from the bitstream back into Vivado for timing checks, etc.
+make dram_test_64x1d_vivado
 ```
```

Comment:
Is it worth mentioning the high level test cases under `tests/` ?

Reply:
I'll add a tests sections in a follow up PR.